src/storage/v3/schemas.hpp

```diff
@@ -25,11 +25,22 @@
 
 namespace memgraph::storage::v3 {
 
+std::string SchemaTypeToString(common::SchemaType type);
+
 struct SchemaProperty {
   PropertyId property_id;
   common::SchemaType type;
 
   friend bool operator==(const SchemaProperty &lhs, const SchemaProperty &rhs);
+
+  friend std::ostream &operator<<(std::ostream &in, const SchemaProperty &schema_property) {
+    in << "SchemaProperty { property_id: ", in << (int)schema_property.property_id.AsUint();
```

Comment:
```suggestion
    in << "SchemaProperty { property_id: ", in << schema_property.property_id.AsUint();
```

This is "C-style" cast, which is a bad practice in C++. If you need to cast the `uint64_t` number to an integer (for the first glance I don't see why it is necessary, but I might be wrong), then use `static_cast<int>(schema_property.property_id.AsUint())` instead, or if `int64_t` is enough, then `schema_property.property_id.AsInt()`.

Reply:
after testing, I've confirmed what you said about it only being weird for byte-sized types, so I'll remove all casting here!